### PR TITLE
CFE-2855 Also list  packages updates for hold packages

### PR DESCRIPTION
**Pull request by basvandervlies**
See:
 * https://tracker.mender.io/browse/CFE-2855

Changelog: Title

### CODE REVIEW

**Comment by nickanderson** [comment]
Are the simulate and ignore-hold options version specific?

**Comment by basvandervlies** [comment]
`--simulate` is the long option instead if the short `-s`. I prefer the long options in scripts for readability.  To get all available updates you must add the option `--ignore-hold`. We use this version in our environment.

File: modules/packages/apt_get [hunk 1]
```diff
@@ -166,7 +166,7 @@ def list_updates(online):
         if result != 0:
             return result
 
-    process = subprocess_Popen([apt_get_cmd] + apt_get_options + ["-s", "upgrade"], stdout=subprocess.PIPE)
+    process = subprocess_Popen([apt_get_cmd] + apt_get_options + ["--simulate", "--ignore-hold", "upgrade"], stdout=subprocess.PIPE)
```
Comment (vpodzime) [inline modules/packages/apt_get]: :+1: for using the long option! I think the `--ignore-hold` option deserves a comment though.

**Comment by nickanderson** [comment]
Superceeded by https://github.com/cfengine/masterfiles/pull/1148 which includes a comments in the code as requested by @vpodzime

**Pull request closed without merging**